bin/plugin/cli.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+// Dependencies
+const program = require( 'commander' );
+const { releaseRC, releaseStable } = require( './commands/release' );
+const { prepublishNpmStablePackages } = require( './commands/packages' );
+
+program
+	.command( 'release-plugin-rc' )
+	.alias( 'rc' )
+	.description(
+		'Release an RC version of the plugin (supports only rc.1 for now)'
+	)
+	.action( releaseRC );
+
+program
+	.command( 'release-plugin-stable' )
+	.alias( 'stable' )
+	.description( 'Release a stable version of the plugin' )
+	.action( releaseStable );
+
+program
+	.command( 'prepublish-packages-stable' )
+	.alias( 'npm-stable' )
+	.description(
+		'Prepublish to npm steps for the next stable version of WordPress packages'
+	)
+	.action( prepublishNpmStablePackages );
+
+program.parse( process.argv );
```

Comment:
Follow-up idea: It would be nice if the CLI displayed _something_ when not called with any arguments.

Reply:
I thought about a similar thing for `wp-scripts`, in case you don't provide the name, you could list all commands with the inquirer and let pick one of the options. More or less what Jest does in the watch mode.

Reply:
There's actually an automated help command `-h` but I didn't find how to easily display it when no  command is passed